sample/src/jsMain/kotlin/Main.kt

```diff
@@ -31,7 +31,7 @@ private fun setupPieChart() {
         val input = document.getElementById(id) as HTMLInputElement
         input.min = min.toString()
         input.max = max.toString()
-        input.step = "0.001"
+        input.step = ((max - min) / 1000f).toString()
```

Comment:
Semi unrelated change. When I went to test on web, I couldn't actually move the slider for end angle close enough to 2 pi to trigger the full circle. This makes the step size variable such that the last step should be exactly max.